src/entity2input.js

```diff
@@ -1,19 +1,21 @@
 const { camelCase, upperFirst } = require('lodash')
-const { entityField2gql } = require("./helpers/gqlConverters")
+const { entityField2gql, EnumConventions } = require("./helpers/gqlConverters")
 const { checker } = require('suma')
 const { entityValidator } = require('./herbsValidator')
 
-function entity2input(entity, isCamelCase = true, customName ) {
+function entity2input(entity, options = undefined) {
```

Comment:
why ` = undefined` if this is the default?

Reply:
maybe ` = {}` which means a empty option but not `null`

Reply:
resolved and remove undefined